File: plugins/elixir-kit/testdata/bad/smell.ex
# Intentional smells for elixir-rg-gate discrimination (not product code).
defmodule Smell do
  def atom_bad(raw), do: String.to_atom(raw)

  def sql_bad(id) do
    q = "SELECT * FROM users WHERE id = #{id}"
    q
  end

  def sleep_bad do
    Process.sleep(1000)
    :ok
  end
end

File: plugins/mdx-kit/testdata/bad/smell.mjs
// Intentional smells for mdx-rg-gate discrimination (not product code).
import {evaluate} from '@mdx-js/mdx'
import rehypeRaw from 'rehype-raw'
import * as runtime from 'react/jsx-runtime'

export async function renderUser(body) {
  return evaluate(body, {...runtime})
}

export const plugins = [rehypeRaw]
